src/main/java/com/example/demo/friend/service/FriendShipService.java

```diff
@@ -21,11 +20,12 @@
 
     private final FriendShipRepository friendShipRepository;
     private final UserRepository userRepository;
+    private final UserSessionService userSessionService;
 
 
     @Transactional
-    public List<FriendDto> getMyFriends(int page, int size) {
-        User user = null;
+    public List<FriendDto> getMyFriends(int page, int size) throws AccessDeniedException {
+        User user = userSessionService.getLoggeddUser();
```

Comment:
User가 강하게 결합되어있네요.
요거는 요렇게 더 일반화 시킬 수 있지 않을까요?

```suggestion
    public List<FriendDto> getFriendsByUserId(Long userId, int page, int size) {
    
```